src/components/activities/RosterPanel.tsx

```diff
@@ -6,6 +6,8 @@ import { useTheme } from '@mui/material/styles';
 import { FunctionComponent, ReactNode, useEffect, useState } from 'react';
 
 import { Box, Dialog, DialogContent, DialogTitle, Paper, Stack, Typography, useMediaQuery } from '@respond/components/Material';
+import { apiFetch } from '@respond/lib/api';
+import { MemberInfo } from '@respond/lib/server/memberProviders/memberProvider';
```

Comment:
Client side code is importing server side classes. This should be avoided. One reason is that the server-side class may have sensitive information that is not intended to be exposed to any client. `/lib/types` would be a better place for the class/interface.

Reply:
created a separate client side interface